src/ontology/edits/oeo-shared.omn

```diff
@@ -1954,7 +1954,7 @@ issue: https://github.com/OpenEnergyPlatform/ontology/issues/1917
 pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1931",
         <http://purl.obolibrary.org/obo/IAO_0000115> "OBSOLETE: An economic value is a quantity value that is economically relevant.",
         <http://purl.obolibrary.org/obo/IAO_0000118> "wirtschaftlicher Wert"@de,
-        rdfs:comment "This term was made obsolete because its definition is rather vague and thus not useful.",
+        rdfs:comment "This term was made obsolete together with economic value because its definition is rather vague and thus not useful.",
```

Comment:
This change imho does not make sense. It basically says that `economic value` was made obsolete with `economic value`.

Maybe you wanted to do this change not at economic value but at the obsolete object properties instead?

Reply:
Oops, thats right....